setup.py

```diff
@@ -292,6 +292,7 @@ def main():
             "Programming Language :: Python :: 3.8",
             "Programming Language :: Python :: 3.9",
             "Programming Language :: Python :: 3.10",
+            "Programming Language :: Python :: 3.11",
```

Comment:
drop line 290 with python 3.6?

Reply:
The package is build from sources for python 3.6. I do not see strong reason to drop it for now.